content/docs/overview.md

```diff
@@ -2672,6 +2672,10 @@ Your .odin files can have a magic suffix that will cause the compiler to either
 
 For example, `foobar_windows.odin` would only be compiled on Windows, `foobar_linux.odin` only on Linux, and `foobar_windows_amd64.odin` only on Windows AMD64.
 
+Another reserved suffix is `_test`. Assume a package with `foobar.odin` and `foobar_test.odin`. Anything in `foobar_test.odin` will be hidden from `foobar.odin`, and only be built and run using the `odin test` command. Note that the entry point procedure `main` must not be located in a file with the `_test` suffix.
```

Comment:
foobar.odin can actually see everything inside foobar_test.odin the same way it can see things in foobar_windows.odin, it's just that the file gets ignored when not building under odin test

Reply:
I tested this by creating procedures and package scope variables inside `foobar_test.odin`. Accessing them from `foobar.odin` gives `Undeclared name` errors.

Reply:
that being said it'd be pretty silly to do something like my example 😆